### PR TITLE
toml: stubbed main functions + tests

### PR DESCRIPTION
**Pull request by ear7h**
I mostly copied the test running function from the `dhall-json` library. It's currently commented out because the functions aren't implemented.

### CODE REVIEW

File: dhall-toml/src/Dhall/Toml.hs [hunk 1]
```diff
@@ -1,35 +1,38 @@
-{-# LANGUAGE OverloadedStrings #-}
-
 module Dhall.Toml
-    ( tomlToDhall
+    ( tomlToDhallMain
+    , tomlToDhall
```
Comment (sjakobi) [inline dhall-toml/src/Dhall/Toml.hs:3]: I think these functions should eventually be moved to a separate module (e.g. `Dhall.TomlToDhall`) but this doesn't necessarily have to happen now.

File: dhall-toml/tasty/data/empty.dhall [hunk 2]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (sjakobi) [inline dhall-toml/tasty/data/empty.dhall:1]: We should probably focus on converting Dhall _values_ first. ;)

```suggestion
{=}
```

File: dhall-toml/src/Dhall/Toml.hs (hunk 1 above)
Comment (ear7h) [reply dhall-toml/src/Dhall/Toml.hs:3]: Okay, I'll add a "TODO" for that

File: dhall-toml/tasty/data/empty.dhall (hunk 2 above)
Comment (ear7h) [reply dhall-toml/tasty/data/empty.dhall:1]: That's a good idea! I've only understood the difference as I've started digging into the internals

**Review by sjakobi — APPROVED** [review]

**Pull request merged**